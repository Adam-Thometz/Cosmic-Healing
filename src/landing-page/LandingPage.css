.LandingPage {
  margin: 5% 0;
}