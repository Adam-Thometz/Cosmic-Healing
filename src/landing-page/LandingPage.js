import React from 'react';

import Hero from './hero/Hero';
import MeetProfessional from './meet-professional/MeetProfessional';
import CanHelpWith from './can-help-with/CanHelpWith';
import Methodology from './methodology/Methodology';
import ServiceInfo from './services-info/ServiceInfo';
import HowItWorks from './how-it-works/HowItWorks';
import Affiliates from './affiliates/Affiliates';

const LandingPage = () => {
  return (
    <main className='LandingPage'>
      <Hero />
      <MeetProfessional />
      <CanHelpWith />
      <Methodology />
      <ServiceInfo />
      <HowItWorks />
      <Affiliates />
    </main>
  );
};

export default LandingPage;