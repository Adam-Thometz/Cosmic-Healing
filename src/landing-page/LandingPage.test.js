import React from "react";
import { render } from "@testing-library/react";
import LandingPage from './LandingPage';

describe('LandingPage component', () => {
  it('renders without crashing', () => {
    render(<LandingPage />);
  });

  it('matches the snapshot', () => {
    const { asFragment } = render(<LandingPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});