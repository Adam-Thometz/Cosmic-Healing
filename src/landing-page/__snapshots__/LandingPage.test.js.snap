// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage component matches the snapshot 1`] = `
<DocumentFragment>
  <main
    class="LandingPage"
  >
    <header
      class="Hero"
      style="--i: 1;"
    >
      <section
        class="Hero-text-wrapper"
      >
        <h1>
          Break the cycle, become a better you
        </h1>
        <p>
          Start your journey today to be on your path to become the person you were always designed to be.
        </p>
        <div
          class="Hero-link-wrapper"
        >
          <a
            class="Link primary"
          >
            Browse services
          </a>
          <small>
            Licensed clinician with over 6 years experience delivering results to over 100 clients
          </small>
        </div>
      </section>
      <aside
        class="Hero-img-wrapper"
      >
        <img
          alt=""
          src="miriam.png"
        />
      </aside>
    </header>
    <section
      class="MeetProfessional"
      style="--i: 2;"
    >
      <aside
        class="MeetProfessional-img-wrapper"
      >
        <img
          alt=""
          src="miriam2.png"
        />
      </aside>
      <aside
        class="MeetProfessional-text-wrapper"
      >
        <img
          alt=""
          class="MeetProfessional-stars"
          src="stars.png"
        />
        <small
          class="SmallCaption"
          style="color: black;"
        >
          MEET THE LICENSED PROFESSIONAL
        </small>
        <h2
          class="SectionHeader"
        >
          Hello, my name is Miriam!
        </h2>
        <article>
          I am a licensed social worker and spiritual life coach who strives to help you uncover your purpose. I use a critically conscious lens, heart-centered approach and astrology charts to align with your higher authentic self. Let's go on this journey together as we sort through and make sense of each puzzle piece to form a full picture.
        </article>
        <a
          class="Link secondary"
        >
          Read more
          <img
            alt=""
            class="Link-arrow"
            src="arrow.png"
          />
        </a>
      </aside>
    </section>
    <section
      class="CanHelpWith"
      style="--i: 3;"
    >
      <aside
        class="CanHelpWith-img-wrapper"
      >
        <img
          alt=""
          src="hand.png"
        />
      </aside>
      <aside
        class="CanHelpWith-text-wrapper"
      >
        <small
          class="SmallCaption"
          style="color: black;"
        >
          Understand yourself
        </small>
        <h2
          class="SectionHeader"
        >
          Learn how to
        </h2>
        <ul
          class="CanHelpWith-slider-wrapper"
        >
          <li
            class="CanHelpWith-slider-text"
          >
            increase emotional intelligence
          </li>
          <li
            class="CanHelpWith-slider-text"
          >
            improve motivation and focus
          </li>
          <li
            class="CanHelpWith-slider-text"
          >
            strengthen relationships
          </li>
          <li
            class="CanHelpWith-slider-text"
          >
            boost self esteem
          </li>
        </ul>
      </aside>
    </section>
    <section
      class="Methodology"
    >
      <small
        class="SmallCaption"
        style="color: black;"
      >
        HOW I CAN HELP..
      </small>
      <h2
        class="SectionHeader"
      >
        My Process
      </h2>
      <div
        class="Methodology-methods"
      >
        <figure
          class="Method"
        >
          <img
            alt="Dreamwork"
            src="dreamwork.png"
          />
          <figcaption>
            <h3>
              Dreamwork
            </h3>
            <p>
              Decode symbols within your dreams to uncover subconscious messages
            </p>
          </figcaption>
        </figure>
        <figure
          class="Method"
        >
          <img
            alt="Shadow Work"
            src="shadow-work.png"
          />
          <figcaption>
            <h3>
              Shadow Work
            </h3>
            <p>
              Integrate the repressed and hidden parts of you
            </p>
          </figcaption>
        </figure>
        <figure
          class="Method"
        >
          <img
            alt="Emotional Processing"
            src="emotional-processing.png"
          />
          <figcaption>
            <h3>
              Emotional Processing
            </h3>
            <p>
              Understand and process your emotions from the past and present
            </p>
          </figcaption>
        </figure>
        <figure
          class="Method"
        >
          <img
            alt="Motivational Interviewing"
            src="motivational-interviewing.png"
          />
          <figcaption>
            <h3>
              Motivational Interviewing
            </h3>
            <p>
              Find internal motivation to change self-destructive behavior
            </p>
          </figcaption>
        </figure>
        <figure
          class="Method"
        >
          <img
            alt="Cognitive Behavioral"
            src="cognitive-behavioral.png"
          />
          <figcaption>
            <h3>
              Cognitive Behavioral
            </h3>
            <p>
              Release negative patterns and limiting beliefs
            </p>
          </figcaption>
        </figure>
        <figure
          class="Method"
        >
          <img
            alt="Person Centered"
            src="person-centered.png"
          />
          <figcaption>
            <h3>
              Person Centered
            </h3>
            <p>
              Gain the confidence to take charge of your own life
            </p>
          </figcaption>
        </figure>
      </div>
    </section>
    <section
      class="ServiceInfo"
      style="--i: 5;"
    >
      <div
        class="ServiceInfo-header"
      >
        <small
          class="SmallCaption"
          style="color: black;"
        >
          DO WHAT IS BEST FOR YOU
        </small>
        <h2
          class="SectionHeader"
        >
          Choose your ideal environment
        </h2>
      </div>
      <div
        class="ServiceInfo-options"
      >
        <div
          class="ServiceCard"
        >
          <h3
            class="SectionHeader"
          >
            Individual Quest
          </h3>
          <article
            class="ServiceCard-description"
          >
            Solo journey.
          </article>
          <a
            class="Link secondary"
          >
            Learn more
            <img
              alt=""
              class="Link-arrow"
              src="arrow.png"
            />
          </a>
        </div>
        <div
          class="ServiceCard"
        >
          <h3
            class="SectionHeader"
          >
            Group Sessions
          </h3>
          <article
            class="ServiceCard-description"
          >
            Go on a collective journey with others.
          </article>
          <a
            class="Link secondary"
          >
            Learn more
            <img
              alt=""
              class="Link-arrow"
              src="arrow.png"
            />
          </a>
        </div>
        <div
          class="ServiceCard"
        >
          <h3
            class="SectionHeader"
          >
            Facts and Questions
          </h3>
          <article
            class="ServiceCard-description"
          >
            Learn more about the practice.
          </article>
          <a
            class="Link secondary"
          >
            Learn more
            <img
              alt=""
              class="Link-arrow"
              src="arrow.png"
            />
          </a>
        </div>
      </div>
    </section>
    <section
      class="HowItWorks"
      style="--i: 6;"
    >
      <aside
        class="HowItWorks-text-wrapper"
      >
        <small
          class="SmallCaption"
          style="color: black;"
        >
          HOW IT WORKS..
        </small>
        <h2
          class="SectionHeader"
        >
          Start in three easy steps
        </h2>
        <ol>
          <li>
            <h3
              class="SectionHeader"
            >
              Motivational Interviewing
            </h3>
            <p>
              Interaction design is the practice of designing interactive digital products, environments, systems, and services.
            </p>
          </li>
          <li>
            <h3
              class="SectionHeader"
            >
              Motivational Interviewing
            </h3>
            <p>
              Interaction design is the practice of designing interactive digital products, environments, systems, and services.
            </p>
          </li>
          <li>
            <h3
              class="SectionHeader"
            >
              Motivational Interviewing
            </h3>
            <p>
              Interaction design is the practice of designing interactive digital products, environments, systems, and services.
            </p>
          </li>
        </ol>
        <a
          class="Link primary"
        >
          Get started
        </a>
      </aside>
      <aside
        class="HowItWorks-img-wrapper"
      >
        <img
          alt=""
          src="paper-plane.png"
        />
      </aside>
    </section>
    <section
      class="Affiliates"
      style="--i: 7;"
    >
      <aside
        class="Affiliates-text"
      >
        <small
          class="SmallCaption"
          style="color: black;"
        >
          PROUD TO BE WORKING IN PARTNERSHIP WITH..
        </small>
      </aside>
      <aside
        class="Affiliates-icons"
      >
        <img
          alt="Inclusive Therapists"
          class="Affiliates-icon"
          src="inclusive-therapists.png"
        />
        <img
          alt="Eastern Group Psychotherapy Society"
          class="Affiliates-icon"
          src="egps.png"
        />
        <img
          alt="National Queer and Trans Therapists of Color Network"
          class="Affiliates-icon"
          src="nqttcn.png"
        />
        <img
          alt="Manhattan Alternative"
          class="Affiliates-icon"
          src="manhattan-alternative.png"
        />
        <img
          alt="Alma"
          class="Affiliates-icon"
          src="alma.png"
        />
        <img
          alt="Therapy for Black Girls"
          class="Affiliates-icon"
          src="therapy-for-black-girls.png"
        />
      </aside>
    </section>
  </main>
</DocumentFragment>
`;
